ui/app/rif/components/searchDomains.js

```diff
@@ -10,8 +10,13 @@ class SearchDomains extends Component {
 			let existDomain = domains.find(domain => domain.domain === e.target.value.toLowerCase())
 			if(existDomain)
 				return this.props.showDomainsDetailPage(existDomain)
-
-			//Here goes the logic to search domains that are not in localstorage
+			//Checks if the domain is available, so if it is, it need to render a screen so the user can register it
+			this.props.checkDomainAvailable(e.target.value.toLowerCase()).then(ret => {
```

Comment:
I think is the same, i mean is just a matter of taste, he can do that but it will need to make handlekeydown async, i think us not necesary but again i think is just a matter of taste here.